src/Routing/Loader.php

```diff
@@ -40,6 +45,10 @@ public function load($resource, $type = null) : RouteCollection
         return $routes->build();
     }
 
+    /**
+     * @param mixed $resource
```

Comment:
the purpose seems to be "declare all types"? But doesn't seem to add information as `$resource` can be any type?

Reply:
Like https://github.com/libero/content-api-bundle/pull/18#discussion_r246318533, this is on the parent so shouldn't be needed.